Client/mods/deathmatch/logic/CClientVehicle.cpp

```diff
@@ -2825,10 +2827,21 @@ void CClientVehicle::Create()
             if (m_bHasCustomHandling)
                 ApplyHandling();
         }
+
         // Re-add all the upgrades - Has to be applied after handling *shrugs*
+        bool bPreviousWheelScaleChanged = m_bWheelScaleChanged;
+        float fPreviousWheelScale = m_fWheelScale;
```

Comment:
```suggestion
        // Applying upgrades can change the wheel scale, so keep track of the original values
        bool bPreviousWheelScaleChanged = m_bWheelScaleChanged;
        float fPreviousWheelScale = m_fWheelScale;

        // Re-add all the upgrades - Has to be applied after handling *shrugs*
```

Reply:
Thank you! I've just commited a variant of your suggested change, changing the comments a bit to make them more clear and concise.